zScanner/Coordinators/AppCoordinator.swift

```diff
@@ -23,8 +23,8 @@ class AppCoordinator: Coordinator {
     
     // MARK: Navigation methods
     private func showSplashScreen() {
-        let viewController = SeaCatSplashViewController(coordinator: self)
```

Comment:
We don't want to commit these changes. You can keep them on your local machine, but don't commit them

Reply:
I did it so I wouldn't all the time when creating a new branch from dev make these changes again. So I'll put it in stashes.